student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age:, name:, classroom:, parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] This would return errors. In the Person class, you defined your parameters using like this: `initialize(age, name = 'Unknown', parent_permission: true)`, but here you are introducing the colon notation. Kindly refactor your definition here to match what you expect in the Person class, like so:

  ```
  class Student < Person
    def initialize(age, classroom, name = "Unknown", parent_permission: true)
      super(age, name, parent_permission: parent_permission)
      ...
  ```
  Note that I defined `classroom` before the `name`, `parent_permission`. Usually, to avoid errors, optional parameters come last.